ambry-cloud/src/main/java/com/github/ambry/cloud/azure/AzureBlobDataAccessor.java

```diff
@@ -90,35 +92,34 @@ public AzureBlobDataAccessor(CloudConfig cloudConfig, AzureCloudConfig azureClou
 
   /**
    * Test constructor
-   * @param blobServiceClient the {@link BlobServiceClient} to use.
-   * @param blobBatchClient the {@link BlobBatchClient} to use.
+   * @param blobServiceAsyncClient the {@link BlobServiceAsyncClient} to use.
+   * @param blobBatchAsyncClient the {@link BlobBatchAsyncClient} to use.
    * @param clusterName the cluster name to use.
    * @param azureMetrics the {@link AzureMetrics} to use.
    * @param azureCloudConfig {@link AzureCloudConfig} object.
    */
-  AzureBlobDataAccessor(BlobServiceClient blobServiceClient, BlobBatchClient blobBatchClient, String clusterName,
-      AzureMetrics azureMetrics, AzureCloudConfig azureCloudConfig) {
+  AzureBlobDataAccessor(BlobServiceAsyncClient blobServiceAsyncClient, BlobBatchAsyncClient blobBatchAsyncClient,
+      String clusterName, AzureMetrics azureMetrics, AzureCloudConfig azureCloudConfig) {
     this.blobLayoutStrategy = new AzureBlobLayoutStrategy(clusterName);
     try {
-      this.storageClient =
-          Utils.getObj(AzureCloudConfig.DEFAULT_AZURE_STORAGE_CLIENT_CLASS, blobServiceClient, blobBatchClient,
-              azureMetrics, blobLayoutStrategy, azureCloudConfig);
+      this.storageClient = Utils.getObj(AzureCloudConfig.DEFAULT_AZURE_STORAGE_CLIENT_CLASS, blobServiceAsyncClient,
+          blobBatchAsyncClient, azureMetrics, blobLayoutStrategy, azureCloudConfig);
     } catch (ReflectiveOperationException roEx) {
       throw new IllegalArgumentException("Unable to instantiate storage client: " + roEx.getMessage(), roEx);
     }
     this.azureMetrics = azureMetrics;
     this.purgeBatchSize = AzureCloudConfig.DEFAULT_PURGE_BATCH_SIZE;
-    requestTimeout = null;
-    uploadTimeout = null;
-    batchTimeout = null;
+    requestTimeout = Duration.ofMillis(CloudConfig.DEFAULT_CLOUD_REQUEST_TIMEOUT);
```

Comment:
Same comment for these three lines -- we should use configs like `CloudConfig.cloudRequestTimeout`